triggers.md

```diff
@@ -272,6 +272,12 @@ An example of using these would be the following, which marks up as safe-to-prer
 
 Note how this example uses a low `"score"` value to indicate that, although these links are _safe_ to prerender, they aren't necessarily that important or likely to be clicked on. In such a case, the browser would likely use its own heuristics, e.g. only performing the prerender on pointer-down. Additionally, the web developer might combine this with a higher-scoring rule that indicates which URLs they suspect are likely, which the browser could prerender ahead of time.
 
+For rule sets that are externally fetched, url patterns in `"href_matches"` will be parsed relative to the external resource's url. To parse these patterns relative to the document that fetched the rules instead, `"href_matches"` can be paired with `"relative_to": "document"` like so:
```

Comment:
Instead of "...the document that fetched the rules", I'd maybe say "...the document" or "...the document which contains the link"?

Reply:
Updated to "the document's base url".